packages/suite/src/components/suite/CardWithHeader.tsx

```diff
@@ -1,6 +1,6 @@
 import React from 'react';
 import styled from 'styled-components';
-import { variables } from '@trezor/components';
+import { variables, Card as CardComponent } from '@trezor/components';
```

Comment:
can we use just `Card`?

Reply:
Yes, forgot to remove after renaming to `CardWithHeader.tsx`

Reply:
done